cf-editor-form-field-dropdown-sample/src/aem-cf-editor-1/web-src/src/components/Dropdown.js

```diff
@@ -54,21 +61,27 @@ export function  Dropdown() {
       <div className={'dropdown-field-wrapper'}>
         <ComboBox
           label={model.fieldLabel}
+          errorMessage={error}
+          loadingState={loading}
+          isReadOnly={!!loading || !items.length || !!error}
```

Comment:
Can this be simpler?
```suggestion
          isReadOnly={loading || !items.length || error}
```